spring-boot/admin/src/main/resources/templates/admin/fragments/addOrderingToList.html

```diff
@@ -0,0 +1,66 @@
+<html xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity4"
+      xmlns="http://www.w3.org/1999/xhtml">
+<body>
+
+<div th:fragment="addOrderingToPublicServicesList">
+    <script>
+        $(document).ready(function () {
+            $('table[class*="default-data-table"]').DataTable({});
+        });
+    </script>
+</div>
+
+<div th:fragment="addOrderingToAdminApiKeyList">
+    <script>
+        $(document).ready(function () {
+            $('table[class*="default-data-table"]').DataTable({
+                columnDefs: [
+                    {orderable: false, targets: [5, 6]}
+                ],
+                "scrollX": true
+            });
+        });
+    </script>
+</div>
+
+<div th:fragment="addOrderingToConsumerApiKeyList">
```

Comment:
Same comment as for previous fragment, maybe unify and parametrize? This also applies to the next two.

Reply:
Done. See comment above.